lib/wallet/src/Cardano/Wallet/Write/Tx/Size.hs

```diff
@@ -0,0 +1,30 @@
+module Cardano.Wallet.Write.Tx.Size
+    ( sizeOfSignedWithdrawal
+    ) where
+
+import Prelude
+
+import Numeric.Natural
+    ( Natural )
+
+sizeOfSignedWithdrawal :: Natural
```

Comment:
Hi @Anviking 

Obviously this is just a draft, so take my comments with a piece of salt!

Having said that, would it be possible to use a more descriptive type than `Natural` here?

Ideally, I think we want something that indicates this is a size, and preferably also the units.

There is already the `TxSize` type, so we could just reuse that.

```suggestion
sizeOfSignedWithdrawal :: TxSize
```